components/server/.env

```diff
@@ -39,3 +39,7 @@ REPLY_SECRET=ThisIsASecretToken
 REFRESH_INTERVAL=15000 # Milliseconds
 
 # Info Server ?
+
+# Optional: retry get_object request config
+# MAX_RETRIES=10
+# RETRY_TIMEOUT=2 # Milliseconds
```

Comment:
I see no point in commenting the config entries out even if there are defaults defined.

```suggestion
# Optional: Retry config (currently only implemented for get_object functionality)
MAX_RETRIES=10
RETRY_TIMEOUT=2 # Milliseconds. Doubles with each re-try.
```